### PR TITLE
doc: Add doc for connecting to ClickHouse Cloud

### PR DESCRIPTION
**Pull request by disq**
Had trouble finding out about the `secure` param (and the error msg isn't helpful) as it's buried deep in:
> See [SDK documentation](https://github.com/ClickHouse/clickhouse-go#dsn) for more details.

### CODE REVIEW

File: plugins/destination/clickhouse/docs/overview.md [hunk 1]
```diff
@@ -96,6 +96,13 @@ spec:
       - "{replica}"
 ```
 
+### Connecting to ClickHouse Cloud
+
+To connect to [ClickHouse Cloud](https://clickhouse.com/cloud), you need to set the `secure=true` parameter, and the port is `9440`. Use a connection string similar to:
```
Comment (candiduslynx) [inline plugins/destination/clickhouse/docs/overview.md:101]: ref https://clickhouse.com/docs/en/cloud-quick-start#5-using-the-clickhouse-client as well?

Comment (candiduslynx) [reply plugins/destination/clickhouse/docs/overview.md:101]: & https://clickhouse.com/docs/en/integrations/go#connection-settings-1

Comment (candiduslynx) [reply plugins/destination/clickhouse/docs/overview.md:101]: possibly, for the connections_string docs as well (& json schema)

Comment (disq) [reply plugins/destination/clickhouse/docs/overview.md]: added in https://github.com/cloudquery/cloudquery/pull/17477/commits/d2d135f698b85a03545d69276f2f4876c8bf08b7

Comment (disq) [reply plugins/destination/clickhouse/docs/overview.md]: The "See [SDK documentation](https://github.com/ClickHouse/clickhouse-go#dsn) for more details." covers the rest I think, and they can always deep dive further.

**Review by candiduslynx — APPROVED** [review]

**Pull request merged**